PersonApi.Tests/V1/Gateways/DynamoDbGatewayTests.cs

```diff
@@ -164,16 +164,17 @@ public async Task UpdatePersonSuccessfulUpdates()
             // Arrange
             var id = Guid.NewGuid();
             var constructRequest = ConstructRequest(id);
-
             var dbEntity = constructRequest.ToDatabase();
 
             // Act
             await _dynamoDb.SaveAsync(dbEntity).ConfigureAwait(false);
 
             // Assert
             var request = ConstructRequest(constructRequest.Id);
-            var response = await _classUnderTest.UpdatePersonByIdAsync(request).ConfigureAwait(false);
-            response.Should().NotBeNull();
+            request.Surname = "Update";
+            var response = await _classUnderTest.UpdatePersonByIdAsync(constructRequest).ConfigureAwait(false);
+            response.Surname.Should().Be(constructRequest.Surname);
```

Comment:
You're still only validating the call return value, and not the actual data in the database.